micro-rdk/src/common/data_store.rs

```diff
@@ -5,7 +5,7 @@
 use crate::proto::app::data_sync::v1::SensorData;
 use bytes::{Buf, BufMut, BytesMut};
 use prost::{encoding::decode_varint, length_delimiter_len, DecodeError, EncodeError, Message};
-use ringbuf::{ring_buffer::RbBase, Consumer, LocalRb, Producer};
+use ringbuf::{ring_buffer::RbBase, Consumer, LocalRb, Producer, Rb};
```

Comment:
Is `Rb` now used directly somewhere? I didn't quite see where.

Reply:
I don't think so, huh. I wonder why clippy didn't catch it

Reply:
Ah, it's a trait. Needs to be imported in order to call `buffer.capacity()`